app/src/main/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerFragmentPresenter.kt

```diff
@@ -400,6 +401,21 @@ class QuestionPlayerFragmentPresenter @Inject constructor(
     )
   }
 
+  private val keyboardVisibilityListener = ViewTreeObserver.OnGlobalLayoutListener {
+    val diff = binding.root.rootView.height - binding.root.height
+    val threshold = 200 * activity.resources.displayMetrics.density // represents 200 dp
```

Comment:
We should use Android's conversion mechanism for this (i.e. TypedValue).

Reply:
Okay, will fix this.